Trimble.Modus.Components/Controls/Layouts/WrapLayout.cs

```diff
@@ -0,0 +1,252 @@
+using Microsoft.Maui.Layouts;
+using MIView = Microsoft.Maui.IView;
+
+namespace Trimble.Modus.Components.Controls.Layouts;
+
+/// <summary>
+/// A layout that arranges views in a wrapping grid.
+/// Ref: https://github.com/mdc-maui/mdc-maui/blob/master/Material.Components.Maui/Components/Layouts/WrapLayout.cs
+/// </summary>
+public partial class WrapLayout : Layout, IItemsLayout
+{
+
+    public static readonly BindableProperty OrientationProperty =
+     BindableProperty.Create(
+         nameof(Orientation),
```

Comment:
Each file has a different bindable property declaration. Can you please check the CustomButton or TMButton and change this

Reply:
Done